src/main/java/frc/robot/commands/vision/AutonBallGetter.java

```diff
@@ -26,6 +27,11 @@
     boolean isFinished = false;
     int scanTracker = 0;
     int maxTrackScans = 10;
+    private static final double cameraAngleX = RobotContainer.config().getDouble("cameraAngleX");
+    private static final Double cameraResX = RobotContainer.config().getDouble("cameraResX");
+    private static final Double ballGetterAcceptedOffsetBounds = RobotContainer.config().getDouble("ballGetterAcceptedOffsetBounds");
+    private static final Double adjustSpeed = RobotContainer.config().getDouble("adjustSpeed");
```

Comment:
It should be `private static final double`, not `private static final Double`. See the difference? While this is technically valid code, it isn't what we want.